ui/src/components/overview/BuildDownloadComponent.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { Alert, Col} from 'patternfly-react';
+import './BuildDownloadComponent.css';
+var QRCode = require('qrcode.react');
+
+class BuildDownloadComponent extends Component {
+
+    render() {
+        const size = this.props.QRSize || 256;
+        return (
+            <div className="build-download-component">
+                <Col md={12}>
```

Comment:
The wireframe and the finished component are slightly different. See:

![qrcode](https://user-images.githubusercontent.com/22113654/43516331-2f9fce92-957d-11e8-8abb-3a9d627899cf.png)

Id go with 2 `<Col md={6}>` and the elements should lie naturally within them then.